src/invisible/how-tos/advanced-analysis/intrinsics/README.ipynb

```diff
@@ -295,12 +296,12 @@
     }
    ],
    "source": [
-    "original_video_path = \"data/museum/upstairs_1_(viewing)-07da3ea4/2abada29_0.0-70.531.mp4\"\n",
-    "undistorted_video_path = \"data/museum/upstairs_1_(viewing)-07da3ea4/2abada29_0.0-70.531_undistorted.mp4\"\n",
+    "original_video_path = \"data/upstairs_1_(viewing)-07da3ea4/2abada29_0.0-70.531.mp4\"\n",
```

Comment:
While it maybe should be called "example-recording" rather than "museum", I think it makes sense for the recording data to be in a separate folder.